[Docs] add note to translation (#13735)
Sylius/Sylius: docs/book/api/translations.rst

In the shop part of our API translatable entities are translated at the server-side and users get an already translated field in the response.
By default it will be provided in default locale configured on the channel, different locale must be configured in the header (for example `accept-language: de_DE`)

.. warning::

>     Remember that the chosen locale has to be enabled in the channel!

Reviewer: ```suggestion
    Remember that the chosen locale has to be enabled in the channel!

```

>     Remember that you have to have enabled chosen locale on the channel.

Reviewer: ```suggestion
    Remember that the chosen locale has to be enabled in the channel!
```
